variables.tf

```diff
@@ -0,0 +1,26 @@
+variable "prefix" {
+  description = "The prefix used for all resources in this environment"
+  default     = "prod-"
+}
+
+variable "location" {
+  description = "The Azure location where all resources in this deployment should be created"
+  default     = "uksouth"
+}
+
+variable "AUTH_CLIENT_ID" {
+  description = "GitHub client ID for authentication."
+}
+
+variable "AUTH_CLIENT_SECRET" {
+  description = "GitHub client secret for authentication."
+}
+
+variable "SECRET_KEY" {
+  description = "The Azure secret key"
+}
```

Comment:
These should both be marked as `sensitive = true` so they don't get printed in `terraform plan` output.

(Also the SECRET_KEY is for signing Flask's session cookies, not really Azure related)